src/js/panels.js

```diff
@@ -284,21 +284,21 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
      * Text to speech panel
      */
     fluid.defaults("gpii.firstDiscovery.panel.speakText", {
-        gradeNames: ["fluid.prefs.panel", "autoInit"],
+        gradeNames: ["fluid.prefs.panel", "gpii.firstDiscovery.attachTooltip.renderer", "autoInit"],
         preferenceMap: {
             "gpii.firstDiscovery.speak": {
                 "model.speak": "default"
             }
         },
-        modelRelay: {
-            source: "{that}.model",
+        modelRelay: [{
+            source: "{that}.model.speakChoice",
             target: "{that}.model.speak",
             // Setup the backward restriction to prevent the component instantiation writes back to
-            // the central model that ends up wiping out the saved prefs at the page reload.
-            backward: "liveOnly",
+            // the central model that results in wiping out the saved prefs at the page reload.
+            forward: "liveOnly",
             singleTransform: {
                 type: "fluid.transforms.valueMapper",
-                inputPath: "speakChoice",
+                inputPath: "",
```

Comment:
I guess more like the way it had been.

```javascript
        modelRelay: [{
            source: "{that}.model",
            target: "{that}.model.speak",
            // Setup the backward restriction to prevent the component instantiation writes back to
            // the central model that results in wiping out the saved prefs at the page reload.
            forward: "liveOnly",
            singleTransform: {
                type: "fluid.transforms.valueMapper",
                inputPath: "speakChoice",
                options: {
                    "yes": true,
                    "no": {
                        outputValue: false
                    }
                }
            }
}]
```